share/tasks.py

```diff
@@ -44,7 +44,12 @@ def transform(self, raw_id):
         graph = transformer.transform(raw)
 
         if not graph or not graph['@graph']:
-            logger.warning('Graph was empty for %s, skipping...', raw)
+            if not raw.normalizeddata_set.exists():
+                logger.warning('Graph was empty for %s, setting no_change to True', raw)
+                RawDatum.objects.filter(id=raw_id).update(no_change=True)
```

Comment:
Transformers don't compare against the database, so this'll only be hit when the raw datum contains nothing the transformer even recognizes, but not in a way that makes it fail... I think this check belongs where we [save changesets](https://github.com/CenterForOpenScience/SHARE/blob/develop/share/models/change.py#L30)